cassandra/query.py

```diff
@@ -26,7 +26,7 @@
 import warnings
 
 from cassandra import ConsistencyLevel, OperationTimedOut
-from cassandra.util import unix_time_from_uuid1, maybe_add_timeout_to_query
+from cassandra.util import unix_time_from_uuid1, maybe_add_timeout_to_query, utcfromtimestamp
 from cassandra.encoder import Encoder
 import cassandra.encoder
```

Comment:
❓ I see that you added this import in this file, but did not change anything else in the file. Is the import used?

Reply:
removed